src/app/api/rooms/[roomId]/route.ts

```
@@ -1,16 +1,20 @@
-import { RoomParams, RoomWithReview } from '@/types/room';
+import { Room, RoomParams } from '@/types/room';
 
 export async function GET(
   request: NextRequest,
   { params }: { params: RoomParams },
-): Promise<CustomResponse<RoomWithReview | undefined>> {
+): Promise<CustomResponse<Room | undefined>> {
   try {
     const roomId = +params.roomId;
 
+    if (isNaN(roomId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)